tests/phpunit/admin/class-wc-stripe-admin-notices-test.php

```
@@ -1023,4 +1023,133 @@ public function test_hide_notices_dismisses_ece_location_notice() {
+	}
+
+	/**
+	 * Test that dismissing the subscription_detached notice sets the post meta.
+	 *
+	 * @return void
+	 */
+	public function test_hide_notices_dismisses_subscription_detached_notice() {
+		$admin_user = self::factory()->user->create( [ 'role' => 'administrator' ] );
+		wp_set_current_user( $admin_user );
+
+		$subscription = new WC_Subscription();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '1140', column '23'). (undefined)

(MissingImport)